Nos/Views/ThreadView.swift

```diff
@@ -32,6 +32,8 @@ struct ThreadView: View {
             NoteButton(
                 note: root,
                 repliesDisplayType: .count,
+                showLikeCount: false,
+                showRepostCount: false,
```

Comment:
I guess I'm confused about what a ThreadView is. I thought these would both be `true` on a thread.

Reply:
You're right. I confounded it with ThreadRootView, where I did want it to be false. I'm updating this code.